core/common/src/main/java/alluxio/client/file/CacheContext.java

```diff
@@ -28,6 +29,10 @@
   private CacheQuota mCacheQuota = CacheQuota.UNLIMITED;
   /** Used in Prestodb to indicate the cache scope. */
   private CacheScope mCacheScope = CacheScope.GLOBAL;
+
+  /** Used in Prestodb to indicate the hiveContext for a file. */
+  private HiveCacheContext mHiveCacheContext = null;
```

Comment:
Might be out of scope of this PR, I'm just thinking mCacheQuota, mCacheScope and mHiveCacheContext should be final and also should be added as the parameter of the constructor.  @apc999 what do you think?

Reply:
This makes sense to me and we can make default constructor